DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Controllers/RepositoryController.cs

```diff
@@ -54,7 +54,6 @@ public class RepositoryController : BaseApiController
             return Ok(repos);
         }
 
-        [Authorize]
         [Route("api/repos/{repoName}/collaborators")]
```

Comment:
Extract magic strings in activity internal, Security.WebApi magic string extracted in other branch